app/styleguide/wskComponentHandler.js

```diff
@@ -75,6 +75,11 @@ var componentHandler = (function() {
       var registeredClass = findRegisteredClass_(jsClass);
       if (registeredClass) {
         createdComponents_.push(new registeredClass.classConstructor(element));
+        // Call any callbacks the user has registered with this component type.
+        var length = registeredClass.callbacks.length;
+        for (var n = 0; n < length; n++) {
+          registeredClass.callbacks[n](element);
+        }
```

Comment:
@sindresorhus I believe he is referring to [this part](https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml?showone=for-in_loop#for-in_loop) of the JS styleguide.

However, this may be a Google-driven project but I don't believe that means it entails following the Google Code Style. The code style may apply to internal production, but this is a public tool and I believe it may have some leeway in what is allowed. We should be able to use whatever code style we chose for it. Keep in mind, I am not a Googler and I have no clue what rules are in place, just speculating.